File: app/src/main/java/com/example/sasham/testproject/util/PreferencesHelper.java
package com.example.sasham.testproject.util;

import android.content.Context;
import android.content.SharedPreferences;
import android.preference.PreferenceManager;

import com.example.sasham.testproject.Constants;

public class PreferencesHelper {

    public static void setTheme(Context context, int theme) {
        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
        prefs.edit()
                .putInt(Constants.APP_THEME, theme)
                .commit();
    }
    public static int getTheme(Context context) {
        SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(context);
        return prefs.getInt(Constants.APP_THEME, -1);
    }
}

File: app/src/main/java/com/example/sasham/testproject/util/PreferencesHelper.kt
package com.example.sasham.testproject.util

import android.content.Context
import android.content.SharedPreferences
import android.preference.PreferenceManager

import com.example.sasham.testproject.Constants

object PreferencesHelper {

    fun setTheme(context: Context, theme: Int) {
        val prefs = PreferenceManager.getDefaultSharedPreferences(context)
        prefs.edit()
                .putInt(Constants.APP_THEME, theme)
                .commit()
    }

    fun getTheme(context: Context): Int {
        val prefs = PreferenceManager.getDefaultSharedPreferences(context)
        return prefs.getInt(Constants.APP_THEME, -1)
    }

    fun setString(context: Context, key: String, value: String) {
        val prefs = PreferenceManager.getDefaultSharedPreferences(context)
        prefs.edit()
                .putString(key, value)
                .commit()
    }

    fun getString(context: Context, key: String, defValue: String): String? {
        val prefs = PreferenceManager.getDefaultSharedPreferences(context)
        return prefs.getString(key, defValue)
    }
}
